public/main.js

```diff
@@ -52,13 +52,24 @@ function reloadTodoList() {
         todos.forEach(function(todo) {
             var listItem = document.createElement("li");
             var deleteBtn = document.createElement("button");
+            var completeBtn = document.createElement("button");
```

Comment:
The complete button and delete button setup is spread out? Could you group like for like and then factor out a function for creating buttons please?